modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -0,0 +1,86 @@
+= Troubleshooting
+
+Find troubleshooting help in these sections.
+
+* xref:troubleshooting.adoc#checkquotas[**Insufficient Quota**]
+* xref:troubleshooting.adoc#containerspending[**Containers in Pending State**]
+* xref:troubleshooting.adoc#bastionpod[**Using the Bastion Pod**]
+
+[#checkquotas]
+== Check quotas
+
+In some cases, pods can become "unhealthy" and the root cause may be an insufficient quota. You can check quotas in the cloud provider's UI. For example, in the Google Cloud Platform (GCP) console, check for any unhealthy pods in your GKE project. Then in the IAM &amp; Admin section of the GCP console, navigate to Quotas. Look for any reported issues with backend services:
+
+image::gcp-quota-example1.png[GCP Quota]
+
+From the GCP Admin Quotas display:
+
+1. Select the row for the service name that is reporting a quota issue
+2. Click **All Quotas** from the Details column
+3. Check the box for the affected quota, and click **Edit Quota**. 
+4. The dialog indicates: "Enter a new quota limit. Your request will be sent to your service provider for approval." Examine the displayed current value and set a new value. 
+5. Enter a brief request description and click **Next**.
+6. Verify your contact information, and click **Submit Request**.
+
+Notice how in the following example the Backend services quota is set to '5', and we're changing it to '50'. For the K8ssandra deployments (Stargate, cass-operator, Reaper, Medusa, and so on), actually `10` might be a sufficient quota.
+
+image::gcp-quota-example2.png[GCP Backend Quota]
+
+[#containerspending]
+== Containers in pending state 
+
+If your pods are stuck in Pending state after installation, or your cloud provider is warning you about Unschedulable Pods, there are a few ways to work through this.
+
+* If some of your pods start, but others like `pulsar-adminconsole` and `pulsar-grafana` are left in an unschedulable state, you might need to add CPUs to your existing nodes, or add an additional node pool. Luna Streaming requires more resources than Pulsar. 
+
+* To examine a specific pod, use `kubectl describe`. For example, if your `pulsar-bookkeeper-0` pod is not scheduling, use `kubectl describe pods/pulsar-bookkeeper-0` to view detailed output on the pod's state, dependencies, and events. 
+
+* To view log output for a specific pod, use `kubectl logs`. Continuing with the `pulsar-bookkeeper-0` example, to get logs for this pod use `kubectl logs pulsar-bookkeeper-0`. If this pod spans multiple containers, add the flag `--all-containers=true`. 
+
+[#bastionpod]
+== Bastion Pod
+
+The bastion pod deploys with Luna Streaming and allows a user to execute administrative commands in a bash shell of the cluster. This can be a handy troubleshooting tool for testing the health of your cluster. 
```

Comment:
```suggestion
The bastion pod deploys with Luna Streaming and allows a user to execute administrative commands in a bash shell of the cluster. This can be a helpful troubleshooting tool for testing the health of your cluster. 
```